app/views/staff/shared/activities/_projects.html.haml

```diff
@@ -3,6 +3,6 @@
     %h2.govuk-heading-m
       = t("page_content.activity.projects")
     - if policy(:project).create?
-      = form_tag(organisation_activities_path(current_user.organisation), method: "post") do
+      = form_tag(organisation_activity_children_path(activity.extending_organisation, activity), method: "post") do
```

Comment:
Could you add something about this change to the commit message, so we understand the reason for the change?

Reply:
Having discussed this over a Slack call, I interpret my confusion as being caused by this commit doing two things at once:
- removing the ability to choose Project (Level C), like the commit message says
- fixing the bug? misunderstanding? where we were sometimes using the wrong organisation to create the project, which is not mentioned in the commit message (`current_user.organisation` -> `activity.extending_organisation` in here, various other changes in specs)

Is this correct @leeky ?


It's this bugfix? misunderstanding-fix? that I would like to see documented in the commit message, because it might be the only institutional memory we have for this. (Think about you-six-months-from-now, and whether you'd be able to trace your own steps if you had to read this cold during a support shift. ;) )

Reply:
I'll update this tomorrow - I get your point, I think I was avoiding explaining the same changes four times - one for each level!

Reply:
Mostly this is about creating a child activity based on its parent, rather than assuming that `current_user.organisation` will be the same thing.

Reply:
> I'll update this tomorrow - I get your point, I think I was avoiding explaining the same changes four times - one for each level!

Thanks! Replacing the org with extending_org isn't explained at all, much less four times 😅 I know it probably seems "obvious" to you, after spending all that time in the code, that it's symmetrical to the changes done in the other paths, but it's not obvious to me!